docs/source/general/abstract_models.rst

```diff
@@ -1,6 +1,6 @@
-===================
+===============
 Abstract Models
-===================
```

Comment:
thank you so much for this small cleanup.. do more if you spot more

Reply:
Done with whatever I've found in the latest iteration.